CHANGES_NEXT_RELEASE

```diff
@@ -5,3 +5,4 @@ Fix: Changing cygnus port does not work (#1698)
 [cygnus-commons][MongoBackend] Proper processing of mongo errors in create index operations (#1756)
 [cygnus-ngsi] Support for NGSIv2 notifications (#953)
 [cygnus-ngsi] Changed log level for some events (#1768, #1769)
+[cygnus-ngsi] Support for native Postgres and Postgis types (#1780)
```

Comment:
```suggestion
[cygnus-ngsi][PostgisSink, PostGRESQLSink] Support for native Postgres and Postgis types (#1780)
```

(I think it would be more accurate that way)

Reply:
Fixed in [4cddae4](https://github.com/telefonicaid/fiware-cygnus/pull/1784/commits/4cddae49ef6548b5bba3f9a7d6c8ddc8d8cf68aa)